tools/building.py

```diff
@@ -1132,6 +1132,13 @@ def metadce(js_file, wasm_file, minify_whitespace, debug_info):
           'name': 'emcc$export$' + export,
           'reaches': []
         })
+  # Because emitDCEGraph only looks at the main js file, and not the worker
+  # we have to explictly add module exported here that are accessed from the
+  # worker, in order to keep them alive through this DCE.
+  # TODO: Find a more robust way to do this, perhaps by also scanning worker.js
+  # for roots.
+  if Settings.USE_PTHREADS:
+    user_requested_exports.append('_emscripten_tls_init')
```

Comment:
@kripken what do you think of this?   Can you think of a better way do doing this?  OK with me landing this and as is for now?

I couldn't find any other way to keep this symbol alive on the JS side since its only referenced from worker.js.

Reply:
It was being kelp alive previously because it was bring added to user_requested_exports.   However because I added it to EXPORTED_FUNCTIONS that excludes it from user_requested_exports (which only included unexpected exports, and those on the command line).